src/statick_tool/statick.py

```diff
@@ -48,6 +50,10 @@ def main() -> None:  # pragma: no cover
     if parsed_args.timings:
         print(tabulate(timings, headers="keys", tablefmt="pretty"))
 
+    if parsed_args.show_all_tool_versions or parsed_args.show_run_tool_versions:
+        tool_versions = statick.get_tool_versions()
+        print(tabulate(tool_versions, headers="keys", tablefmt="grid"))
```

Comment:
i like this table format `grid` better for tool versions. and I like the `pretty` table format better for timings. otherwise i'd use the same format.

Reply:
The `grid` table format looks good to me.